src/hooks/xcm/useXcmEvm.ts

```diff
@@ -33,7 +34,8 @@ export function useXcmEvm(selectedToken: Ref<Asset>) {
   const currentWallet = computed(() => store.getters['general/currentWallet']);
   const isMoonbeamWithdrawal = computed<boolean>(() => {
     const tokenContractAddress = selectedToken.value.mappedERC20Addr.toLowerCase();
-    return tokenContractAddress === MOVR.address.toLowerCase();
+    const MoonbeamTokens = [MOVR.address.toLowerCase(), GLMR.address.toLowerCase()];
```

Comment:
it might be PascalCase only for being used class or something? camelCase is more good (> moonbeamTokens)

Reply:
Good catch! I've fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/504/commits/5ffeff1f0e3cb47e395b7202474df8b6037ddd2d#diff-aa82075914ca04a74eafda59499d504733c8ab85ddb039b43e5aa33e963cb1b9R37) commit.